CriticalMass/SettingsCheckTableViewCell.swift

```diff
@@ -0,0 +1,20 @@
+//
+// Created for CriticalMaps in 2020
+
+import UIKit
+
+class SettingsCheckTableViewCell: UITableViewCell, IBConstructable {
+    private var switchable: Toggleable?
```

Comment:
switchable or toggleable? hm, kind of confusing...

Reply:
haha. true. actually this is not needed and I have a fix PR already in place #343

Reply:
So I will add a TODO and then clean that up after #343 has landed

Reply:
Updated